src/DrCabal/Profile.hs

```diff
@@ -73,17 +77,6 @@ groupEntries = foldl' insert mempty
         toVal x Nothing   = [x]
         toVal x (Just xs) = x : xs
 
-data Phase = Phase
-    { phaseDownloading :: Word64
-    , phaseStarting    :: Word64
-    , phaseBuilding    :: Word64
-    , phaseHaddock     :: Word64
-    , phaseInstalling  :: Word64
-    }
-
-phaseTotal :: Phase -> Word64
-phaseTotal (Phase p1 p2 p3 p4 p5) = p1 + p2 + p3 + p4 + p5
-
 calculatePhases :: Word64 -> Map Text [(Status, Word64)] -> Map Text Phase
 calculatePhases start = fmap (entriesToPhase start)
```

Comment:
`calculatePhases` is also `Stacked`-specific function. The calculation logic for `Timeline` will be completely different

Reply:
Done.